README.md

```diff
@@ -4,7 +4,7 @@ tutorials:
   - title: Understanding the Group Module
     level: intermediate
     description: Learn what the Group module does, how to create group, and how to submit group proposals.
-    url: /tutorials/understanding-groups
+    url: /tutorials/understanding-group
```

Comment:
Was this not already corrected in another PR?

Reply:
Yes I need to rebase